Common/Sources/Extensions/String.swift

```diff
@@ -74,6 +74,11 @@ public extension String {
             return ""
         }
     }
+
+    /// Convert a string object to a double object, and remove "," in the string object
+    func toDouble() -> Double? {
+       return Double(self.replacingOccurrences(of: ",", with: "", options: .numeric))
```

Comment:
there are many symbols of group exist. e.g.: Ukraine uses space-symbol.

Reply:
For now, we only deal with ",". As you can see from this pr, I removed the locale from amountFormatter in NumberFormatter extension. This is because our system should not apply to the system language, instead, the amount format should follow the currency.  By default, the amount will add "," for large number if we don't provide locale. So, we only need to remove "," for now.

Reply:
for now, ok